packages/widget-chat/src/utils/browser.js

```diff
@@ -31,8 +31,8 @@ export function isNotificationSupported() {
   return window && window.Notification;
 }
 
-export function requestPermissionForNotifications() {
-  return window.Notification.requestPermission();
+export function requestPermissionForNotifications(callback) {
```

Comment:
I'm confused. why the wrapper function rather than just calling Notification.requestPermission directly?